git_intro/git_intro.md

```diff
@@ -152,6 +193,22 @@ Collaborative writing with traditional word processors is cumbersome. Either eve
 </div>
 ***
 
+Which of the following is a true statement?  Check all that apply!
+
+[[ ]] Git and GitHub are related products made by the same company
+[[X]] GitHub makes a visual interface that some Git users prefer over the command line
+[[ ]] GitHub is the only software that embeds Git functionality
+[[X]] GitHub adds tools that make using Git easier for teams
+***
+<div class = "answer">
+
+GitHub is a company with a website and software that use Git.  GitHub is well known and frequently used because it makes a visual interface that some Git users prefer, when compared to the command line, and because GitHub adds its own tools to the basic Git toolkit, especially tools that make using Git easier for teams.  
```

Comment:
```suggestion
GitHub is a company with a website and software that use git.  GitHub is well known and frequently used because it makes a visual interface that some git users prefer, when compared to the command line, and because GitHub adds its own tools to the basic git toolkit, especially tools that make using git easier for teams.  
```